index.js

```diff
@@ -275,6 +275,8 @@ var characterMap = {
 	"ş": "s",
 	"Š": "S",
 	"š": "s",
+	"ṣ": "s",
+	"Ṣ": "S",
```

Comment:
Minor: The pattern we've followed so far has been to have the capital first, then followed by the lowercase. Could we do the same here?

Note that this applies to the tests as well btw.

Reply:
Sure thing, done. :)